vertx-opentelemetry/src/main/asciidoc/index.adoc

```diff
@@ -67,3 +67,12 @@ The default sending policy is `PROPAGATE`, you can configure the policy with {@l
 ----
 {@link examples.OpenTelemetryExamples#ex6}
 ----
+
+== Vert.x tracing and auto-configuration
```

Comment:
If you agree to modify the code as suggested, I think this part should be re-written to indicate that the `VertxContextStorage` is fine with pure Vert.x application but it might not be the best choice for mixed apps. And then the user needs to select the appropriate storage via the sysprop

Reply:
Both we, as the reporter of #62 seem to have seen this in a pure Vert.x applications, but OpenTelemetry itself is also creating traces for the actions it does, and those traces need a `ContextStorage` which works for them.
I will see if I can try to reword it.